cmd/fortifyExecuteScan_test.go

```diff
@@ -637,6 +647,60 @@ func TestTriggerFortifyScan(t *testing.T) {
 	})
 }
 
+func TestGetMinSpotChecksPerCategory1(t *testing.T) {
```

Comment:
These tests are quite valuable. I would suggest to refactor them by factoring out a generic test and then instantiating it multiple times, like
```
test(100, 10, "percentage", 10, 10);
...
test(10, 10, "percentage", 10, 1);
```
and so on.
That would create a table-like structure that is easier to review. 
In any case, we should sort the entries in the map so that keys related to minimum are next to each other.

Reply:
done